Search/Adapter/ZendLuceneAdapter.php

```diff
@@ -185,7 +185,9 @@ public function search(SearchQuery $searchQuery)
 
             $searcher->addIndex($this->getIndex($indexPath, false));
         }
+        if ($searchQuery->getSort() !== null) {
 
+        }
```

Comment:
Currently there is a "Multisearcher" used - this "Multisearcher" does not support sorting.

Any clue how this could be done quickly?